schematicbrushreborn-core/src/main/java/de/eldoria/schematicbrush/util/Database.java

```diff
@@ -0,0 +1,48 @@
+package de.eldoria.schematicbrush.util;
+
+import java.util.Map;
+
+public class Database {
```

Comment:
Using the configuration Serializable like the other config classes do might be better ^^

This might work, but its not clean.